src/components/Reports/DesignationAccountsReport/DesignationAccountsReport.tsx

```diff
@@ -0,0 +1,113 @@
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import { Box, CircularProgress, Divider, styled } from '@material-ui/core';
+import {
+  DesignationAccountsQuery,
+  useDesignationAccountsQuery,
+} from './GetDesignationAccounts.generated';
+import { useSetActiveDesignationAccountMutation } from './SetActiveDesignationAccount.generated';
+import { DesignationAccountsHeader as Header } from './Layout/Header/Header';
+import { DesignationAccountsList as List } from './Layout/List/List';
+import { Notification } from 'src/components/Notification/Notification';
+import { EmptyReport } from 'src/components/Reports/EmptyReport/EmptyReport';
+import { currencyFormat } from 'src/lib/intlFormat';
+
+interface Props {
+  accountListId: string;
+  isNavListOpen: boolean;
+  onNavListToggle: () => void;
+  title: string;
+}
+
+const ScrollBox = styled(Box)(({}) => ({
+  height: 'calc(100vh - 160px)',
+  overflowY: 'auto',
+}));
+
+export const DesignationAccountsReport: React.FC<Props> = ({
+  accountListId,
+  isNavListOpen,
+  onNavListToggle,
+  title,
+}) => {
+  const { t } = useTranslation();
+
+  const { data, loading, error } = useDesignationAccountsQuery({
+    variables: {
+      accountListId,
+    },
+  });
+
+  const [
+    setActiveDesignationAccount,
+  ] = useSetActiveDesignationAccountMutation();
+
+  const handleCheckToggle = (
+    event: React.ChangeEvent<HTMLInputElement>,
+    designationAccount: DesignationAccountsQuery['designationAccounts'][0]['designationAccounts'][0],
+  ) => {
+    setActiveDesignationAccount({
+      variables: {
+        input: {
+          accountListId,
+          active: event.target.checked,
+          designationAccountId: designationAccount.id,
+        },
+      },
+      optimisticResponse: {
+        setActiveDesignationAccount: {
+          ...designationAccount,
+          active: event.target.checked,
+        },
+      },
+    });
+  };
+
+  return (
+    <Box>
+      <Header
+        isNavListOpen={isNavListOpen}
+        onNavListToggle={onNavListToggle}
+        title={title}
+        totalBalance={
+          data &&
+          currencyFormat(
+            data.designationAccounts[0].balance ?? 0,
```

Comment:
This balance needs to update when accounts get checked and unchecked. Idk if you want to do it with some sort of client side code or GraphQL refreshing...